notebooks/sippy-analysis.ipynb

```diff
@@ -0,0 +1,2411 @@
+{
```

Comment:
<p>I'm unclear on this part "...analyzed using the Sippy algorithm" what is the Sippy algorithm?</p><p> </p><p>I see that regex is used to manually categorize many of the tests, but why would that stop us from reproducing that if we have the regex patterns? And I don't think we would need a special regex pattern for all of these, "Job Pass Rates By Job Name" seems to just be the number of times a job passes over the number of times its run grouped by name.</p><p> </p><p>I think we should use the known regex patterns and metrics they've defined to create plots for each of the items you've listed as part of the Sippy work.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>Sure, I will fix this. </p>